broker/plugins/kubejobs/plugin.py

```diff
@@ -233,6 +235,59 @@ def errors(self):
             return ()
         return self.rds.lrange("job:errors", 0, -1)
 
+    def validate(self, data):
```

Comment:
Shouldn't we add this method in broker.plugins.base (i.e. should all plugins validate it's info?)
If so, this could be parametrized.

Reply:
No, because the plugin structure isn't consistent, and since we agreed that we aren't going to do structural changes, this was the best solution.